viewer/src/views/threejs/cad/SectorNode.ts

```diff
@@ -3,14 +3,18 @@
  */
 
 import * as THREE from 'three';
+import { LevelOfDetail } from '../../../data/model/LevelOfDetail';
 
 export class SectorNode extends THREE.Group {
```

Comment:
I am thinking of later having a revealThreeTransformer where you basically get our core types and output Object3D objects. I think that would clear up a bit of these classes.

Reply:
The RootSector has the job of converting a `ParsedSector` (core) to a `ConsumedSector` (three) now. Should we create a separate transformer class that does this?

Also, the class you have commented on here cannot be created live by the transformer. We need a tree of `SectorNode` objects (or just `THREE.Group`) that always exist to avoid rebuilding the tree itself all the time. It allows us to insert a `THREE.Object3D` into the correct place in the hierarchy directly.